weave/legacy/wandb_interface/project_creator.py

```diff
@@ -72,4 +77,6 @@ def _ensure_project_exists(entity_name: str, project_name: str) -> None:
                 raise UnableToCreateProject(
                     f"Failed to create project {entity_name}/{project_name}"
                 )
-    return
```

Comment:
Is there ever a case that we need entity too?

Reply:
in the future we might want to get the project id from here too for client-side digest calcs

Reply:
I'd recommend making a structured return object that we can extend

Reply:
right, on it

Reply:
@tssweeney to get the entity back we are going to have to make a change in the wandb sdk

Reply:
we don't need it right now ... just want to make sure the return type is a class that is structured